### PR TITLE
Bug fixing

### PR DESCRIPTION
**Pull request by edsonarios**
Fix bug video converter
Fix bug audio converter

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 1]
```diff
@@ -97,7 +96,7 @@ public void actionPerformed(final ActionEvent e) {
                     + "\nConvert to: "
                     + audioSelect.getConvertTo()
                     + "\nQuality: "
-                    + quality.getQualityAudio()
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: Maybe these lines should be one line  + "\nConvert to: " + audioSelect.getConvertTo()

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 2]
```diff
@@ -33,16 +34,22 @@
...
 
     protected OutputSettingsAudio() {
+        JLabelStyle qualityLabel = new JLabelStyle("Select Quality: ", "h3",
+                alignLabelStyle, widthLabelStyle, heightLabelStyle);
         JLabelStyle volumeLabel = new JLabelStyle("Select Volume: ", "h3",
                 alignLabelStyle, widthLabelStyle, heightLabelStyle);
         JLabelStyle hzLabel = new JLabelStyle("Select Hz: ", "h3",
                 alignLabelStyle, widthLabelStyle, heightLabelStyle);
         JLabelStyle audioChannelLabel = new JLabelStyle("Select Audio Channel: ", "h3",
                 alignLabelStyle, widthLabelStyle, heightLabelStyle);
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: edson maybe these lines should try to get to the top of 120 characters

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 3]
```diff
@@ -22,6 +22,7 @@
  * This class defines the interface for output settings for audio to convert.
  */
 class OutputSettingsAudio extends JPanel {
+    private JComboBox qualitySelect;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java:25]: constant names must be capitalized

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 4]
```diff
@@ -33,16 +34,22 @@
     private final int dimensionHeight = 30;
     private final int fontStyle = 0;
     private final int fontSize = 12;
```
Comment (jpgonzales27) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: In this file constant names must be capitalized

File: src/main/java/org/fundacion/jala/converter/core/facade/ZipFileFacade.java [hunk 5]
```diff
@@ -71,27 +70,23 @@ private static void getZipFile(final ParameterOutputChecksum parameterOutputChec
...
-                zipList.add(pathFile + nameWithoutExtension + "png");
-            }
-            Thread.sleep(WAIT_TIME);
-            zipFiles(zipList, pathFile + nameWithoutExtension + "zip");
-        } else {
-            Thread.sleep(WAIT_TIME);
-            zipFile(pathFile + outputFilename, pathFile + nameWithoutExtension + "zip");
+        ArrayList<String> zipList = new ArrayList<>();
+        zipList.add(pathFile + storagePath);
+        if (metadata.equals("true")) {
```
Comment (SaulCM) [inline src/main/java/org/fundacion/jala/converter/core/facade/ZipFileFacade.java]: Could be "true".equals(metadata)

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 6]
```diff
@@ -67,12 +76,23 @@ protected OutputSettingsAudio() {
...
 
+    /**
+     * Sets all possible quality for Audio converter.
+     */
+    protected void setQualitySelect() {
+        qualitySelect.addItem("");
+        qualitySelect.addItem("64");
+        qualitySelect.addItem("128");
+        qualitySelect.addItem("192");
+        qualitySelect.addItem("320");
```
Comment (SaulCM) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: Magic numbers, would be better to set them to final attributes, so they are more understandable.

**Review by SaulCM — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 7]
```diff
@@ -38,29 +38,38 @@
...
-    private final int alignLabelStyle = 2;
-    private final int widthLabelStyle = 70;
-    private final int heightLabelStyle = 30;
-    private final int topBorder = 40;
-    private final int leftBorder = 40;
-    private final int bottomBorder = 100;
-    private final int rightBorder = 0;
-    private final int fontStyle = 0;
-    private final int fontSize = 12;
+//    private final int alignLabelStyle = 2;
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: This comments should be ereased

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 8]
```diff
@@ -94,20 +102,13 @@ public void actionPerformed(final ActionEvent e) {
...
-                    + settings.getVolume()
-                    + "\nAudio Channel: "
-                    + settings.getAudioChannel()
-                    + "\nHz: "
-                    + settings.getHz()
-                    + "\nwith metadata: "
-                    + settings.isMetadata()
-                    + "\nChecksum: "
-                    + checksumLocal);
+                    + "\nConvert to: " + audioSelect.getConvertTo()
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: This lines can be reduced, the top is 120 characters

**Review by JorgeRCV — APPROVED** [review]

**Review by jpgonzales27 — APPROVED** [review]

**Pull request merged**